switch-configuration/Makefile

```diff
@@ -32,20 +32,30 @@ build-switch-configs: .build-switch-configs
 
 # Make the switch-maps-bundle which is a collection of all the switch port maps
 # this is a for printing for the cable team
-switch-maps-bundle: config/switch-maps/bundle.ps config/switch-maps/refs.ps config/switch-maps/stickers.ps
+switch-maps-outputs: config/switch-maps/refs.ps config/switch-maps/refs.pdf config/switch-maps/stickers.ps config/switch-maps/stickers.pdf
```

Comment:
Thanks for incorporating the .pdf generation. I added `ghoscript` to the nix devShell so its easy to run this while in that context.